README.md

```diff
@@ -66,7 +66,7 @@ The [conductor crate](conductor) uses the [conductor_api](conductor_api) to impl
 The [nodejs_conductor](nodejs_conductor) directory implements a node package that creates a conductor that wraps the Holochain core Rust implementation so we can access it from node.  This is crucial especially for creating a test-driven development environment for developing Holochain DNA.  The `hc` command-line tool relies on it to run tests.
 
 ### HC Command-line developer tool.
-The [cli crate](cli) implements our command line developer tool which allows you to create DNA scaffold, run tests, and finally package your DNA for running in a containter.  For more details see the [crate README](cli/README.md).
+The [cli crate](cli) implements our command line developer tool which allows you to create DNA scaffold, run tests, and finally package your DNA for running in a conductor.  For more details see the [crate README](cli/README.md).
```

Comment:
Please use Github's suggestion feature for small change requests like this one, @sphinxc0re:
```suggestion
The [cli crate](cli) implements our command line developer tool which allows you to create DNA scaffold, run tests, and finally package your DNA for running in a Conductor.  For more details see the [crate README](cli/README.md).
```